<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>982</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1234567</width>
    <height>1235443</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget
{
	
	background-image: url(:/res/NEU3.png);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Menu">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>1000</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>960</width>
        <height>640</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 12px;
}
QPushButton:hover
{
	color:rgb(255, 255, 255);
}
</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>1000</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 40pt &quot;华文行楷&quot;;
color: rgb(0, 0, 0);

</string>
       </property>
       <property name="text">
        <string>学生成绩管理系统</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>200</y>
         <width>411</width>
         <height>741</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="Listbtn">
          <property name="minimumSize">
           <size>
            <width>269</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>20</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(79, 147, 255);
font: 20pt &quot;华文行楷&quot;;
boder-radius: 50px;</string>
          </property>
          <property name="text">
           <string>学生信息的增删改查</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Findbtn">
          <property name="minimumSize">
           <size>
            <width>269</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(79, 147, 255);
font: 20pt &quot;华文行楷&quot;;</string>
          </property>
          <property name="text">
           <string>学生信息查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Sortbtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(79, 147, 255);
font: 20pt &quot;华文行楷&quot;;</string>
          </property>
          <property name="text">
           <string>学生成绩排序</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Exitbtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(79, 147, 255);
font: 20pt &quot;华文行楷&quot;;</string>
          </property>
          <property name="text">
           <string>退出系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="basicpage">
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>640</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QTableWidget" name="Stu_Info">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1381</width>
         <height>781</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>820</y>
         <width>691</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="add_stubtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>添加学生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_allbtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>保存所有修改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_stubtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>删除学生</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Findpage">
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>90</y>
         <width>591</width>
         <height>701</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="find_idbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按学号查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_namebtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按姓名查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_sexbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按性别查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_folkbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按民族查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_agebtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按年龄查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_addrbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按地址查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_gpabtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>按各门课程成绩查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Searchpage">
      <widget class="QTableWidget" name="id_search">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>1001</width>
         <height>691</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>511</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="keyword">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findbtn">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>642</width>
         <height>40</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;华文行楷&quot;;
color: rgb(79, 255, 135);

</string>
          </property>
          <property name="text">
           <string>选择你查询的特征</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 30px;</string>
          </property>
          <item>
           <property name="text">
            <string>按学号查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按姓名查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按性别查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按民族查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按年龄查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按地址查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按总绩点查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按面向对象程序设计绩点查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按面向对象课设绩点查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按高数绩点查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按线代绩点查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按大物绩点查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按大英绩点查询</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Sortpage">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>831</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 40pt &quot;华文行楷&quot;;
color: rgb(0, 0, 0);

</string>
       </property>
       <property name="text">
        <string>学生成绩排名页面</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>150</y>
         <width>591</width>
         <height>701</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="gpa_sortbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>总绩点排名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="oop_gpabtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>面向对象程序设计绩点排名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="oopds_sortbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>面向对象课设绩点排名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="math_gpabtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>高数绩点排名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="al_sortbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>线代绩点排名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="phy_gpabtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>大物绩点排名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eg_gpabtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(79, 147, 255);
	font: 20pt &quot;华文行楷&quot;;
	boder-radius: 50px;
}
QPushButton:hover {
    background-color: rgb(29, 97, 255);
}</string>
          </property>
          <property name="text">
           <string>大英绩点排名</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Sortgpapage">
      <widget class="QTableWidget" name="Sort_Stu">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1151</width>
         <height>631</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
